core/src/main/scala/org/broadinstitute/dsde/rawls/util/WithSupport.scala

```diff
@@ -62,6 +62,18 @@ trait MethodWiths {
         )
     }
 
+    method.validate match {
+      case Some(_) => fetchMethod
+      case None =>
+        DBIO.failed(
+          new RawlsExceptionWithErrorReport(
+            errorReport =
+              ErrorReport(StatusCodes.BadRequest, s"Invalid method request. One or more values in request are invalid.")
```

Comment:
Feels like we're duplicating the dictionary definition of a 400 response in the message.

Since we haven't touched user data yet, is it valid to tell the user which field is bad? Or is that a bigger change? Is this a standard message or one you made up? I might lean towards leaving it empty or simplifying to "invalid request" or "invalid parameters"

Reply:
No this isn't a standard message. It is hard to specify which fields are invalid here because `MethodRepoMethod` can be one of `DockstoreMethod`, `DockstoreToolsMethod` or `AgoraMethod` and AgoraMethod has different fields from the Dockstore methods. I was a little hesitant to change `.validate` method to return which fields are invalid as it would have a wider change effect. I can leave the error message as `Invalid request` as that would be easier.

Reply:
Sounds good to me 👍